src/lib/tmux-lite/server.ts

```
@@ -1298,26 +1662,37 @@ function createSession(
-Bun.listen({
+routerListener = Bun.listen({
   unix: ROUTER_SOCKET,
   socket: {
     open(socket) {
       const socketState = getRouterSocketState(socket);
       socketState.writer = createBufferedSocketWriter(socket as any);
     },
-    data(socket, data) {
+    close(socket) {
+      clearRouterSocketState(socket);
+    },
```

Biome (2.4.6)

[error] This method is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)